pkg/operator/starter.go

```diff
@@ -101,3 +106,27 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 
 	return fmt.Errorf("stopped")
 }
+
+func withSecretHashAnnotation(kubeClient kubernetes.Interface, namespace, secret string) func(*appsv1.Deployment) error {
+	return func(deployment *appsv1.Deployment) error {
+		inputHashes, err := resourcehash.MultipleObjectHashStringMapForObjectReferences(
+			kubeClient,
+			resourcehash.NewObjectRef().ForSecret().InNamespace(namespace).Named(secret),
+		)
```

Comment:
@jsafrane for some reason the lister doesn't find the secret in GCP, which is strange because the informers were started above. Do you have any idea what could be wrong?

Reply:
apparently the lister isn't working because nobody is syncing the informer cache.